DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -286,17 +296,20 @@ private function loadJackalopeSession(array $session, ContainerBuilder $containe
             $definition->setFactoryMethod('login');
         }
 
+        $workspace = $admin ? 'default' : $session['workspace'];
```

Comment:
and according to the PHPCR spec, the default should be `null`, not the name 'default'. its up to the implementation what the default workspace is named and you specify you want the default by passing null.